ga4gh/datamodel/rna_quantification.py

```diff
@@ -0,0 +1,273 @@
+"""
+Module responsible for translating feature expression data into GA4GH native
+objects.
+"""
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import os
+
+import ga4gh.datamodel as datamodel
+import ga4gh.protocol as protocol
+
+
+"""
+TODO: Would be nice to just use the csv module to read inputs and have headers
+in files for clarity and to eliminate the whole record[N] absurdity.
+Additionally, characterization and read counts doesn't have an access point.
+"""
+
+
+class RNASeqResult(datamodel.DatamodelObject):
+    """
+    Class representing a single RnaQuantification in the GA4GH data model.
+    """
+    compoundIdClass = datamodel.RnaQuantificationCompoundId
+
+    def __init__(self, parentContainer, rnaQuantificationId, rnaQuantDataPath):
+        super(RNASeqResult, self).__init__(parentContainer, rnaQuantificationId)
+        self._rnaQuantificationId = rnaQuantificationId
+        self._rnaQuantificationFile = os.path.join(
+            rnaQuantDataPath, "rnaseq.table")
+        self._characterizationFile = os.path.join(
+            rnaQuantDataPath, "dist.table")
+        self._readCountFile = os.path.join(rnaQuantDataPath, "counts.table")
+        self._expressionLevelFile = os.path.join(
+            rnaQuantDataPath, "expression.table")
+
+    def convertCharacterization(self, record):
+        readCharacterization = protocol.Characterization
+        readCharacterization.analysisId = record[0]
+        readCharacterization.complexity = float(record[1])
+        readCharacterization.exonicFraction = float(record[2])
+        readCharacterization.fractionMapped = float(record[3])
+        readCharacterization.intergenicFraction = float(record[4])
+        readCharacterization.intronicFraction = float(record[5])
+
```

Comment:
I don't think this blank like before the return adds any readability --- I would suggest removing it.